logger/module.go

```diff
@@ -92,15 +97,30 @@ func (m *loggerModule) SetBus(bus modules.Bus) {
 
 func (m *loggerModule) GetBus() modules.Bus {
 	if m.bus == nil {
-		m.logger.Fatal().Msg("Bus is not initialized")
+		m.Logger.Fatal().Msg("Bus is not initialized")
 	}
 	return m.bus
 }
 
 func (m *loggerModule) InitLogger() {
-	m.logger = m.CreateLoggerForModule(m.GetModuleName())
+	m.Logger = m.CreateLoggerForModule(m.GetModuleName())
 }
 
 func (m *loggerModule) GetLogger() modules.Logger {
-	return m.logger
+	return m.Logger
+}
+
+// SetFields sets the fields for the global logger
+func (m *loggerModule) SetFields(fields map[string]interface{}) {
```

Comment:
Inside of `telemetry/prometheus_module.go`, we have a variadic function that takes in any number of labels.

This can be a TODO for a followup, but I think there is an opportunity to overload this function by changing this to `SetFieldsMap` and having `SetFields` accept an even number of any parameters.

You can use the `INVESTIGATE` tag to track this rather than a `TODO` since it's just an idea.